ui/components/DashboardComponent/index.js

```diff
@@ -75,7 +75,9 @@ const ResourceCategoryTabs = ['Overview', ...Object.keys(ResourcesConfig)];
 const DashboardComponent = ({ k8sconfig, selectedK8sContexts, updateProgress }) => {
   const { data: userData } = useGetUserPrefQuery();
   const [updateUserPref] = useUpdateUserPrefMutation();
-  const defaultLayout = userData?.remoteProviderPreferences ? DEFAULT_LAYOUT : LOCAL_PROVIDER_LAYOUT; //TODO: Use capability to determine default layout
+  const defaultLayout = userData?.remoteProviderPreferences
+    ? DEFAULT_LAYOUT
+    : LOCAL_PROVIDER_LAYOUT; //TODO: Use capability to determine default layout
```

Comment:
Can we please revert this change? As it is not related to actual fix (probably your linter in editor was auto triggered)
```suggestion
const defaultLayout = userData?.remoteProviderPreferences ? DEFAULT_LAYOUT : LOCAL_PROVIDER_LAYOUT; //TODO: Use capability to determine default layout
```

Reply:
nvm. I see, it aims to fix failing eslint workflow. https://github.com/meshery/meshery/actions/runs/13344800734/job/37273942047 👍